### DISCUSSION TITLE
Convert issues to discussions

### DISCUSSION THREAD

**Discussion by iamjonny** [community/community#19942] [Mobile]
Hi, am raising as a feature request as I believe this option doesn't exist (on Android Github app, version info available if required)

Tldr: add options to switch issues into discussions and vice versa, like on web UI

Many thanks for the Awesome app 👍

**Comment by elisealix22** [comment]
Hi @iamjonny , thank you so much for the request! I opened a new tracking issue internally and raised it with our team with some screenshots of the web flow. If you have any more ideas please keep them coming!
